packages/core/src/context/experiment/experiment-reducers.ts

```diff
@@ -17,8 +17,8 @@ import { createFetchExperimentResultRequest } from '@core/context/experiment/api
 
 const calculateInitialPoints = (state: ExperimentType) =>
   Math.max(
-    3,
-    (state.categoricalVariables.length + state.valueVariables.length) * 3
+    5,
+    state.categoricalVariables.length + state.valueVariables.length + 1
```

Comment:
Should this be updated to only be enabled variables once that PR is merged?

Reply:
Good point - I will add that after the merge of your PR